libs/deposits/src/lib/deposit-form.spec.tsx

```diff
@@ -250,7 +250,7 @@ describe('Deposit form', () => {
       screen.getByText('Deposit', { selector: '[type="submit"]' })
     );
 
-    await waitFor(() => {
+    waitFor(() => {
```

Comment:
chore (blocking): Pretty sure you must await this function for it to do anything of value.

See: https://testing-library.com/docs/dom-testing-library/api-async/#waitfor